pkg/commands/clusterstack/save.go

```diff
@@ -41,12 +41,11 @@ kp clusterstack create my-stack --build-image ../path/to/build.tar --run-image .
 				return err
 			}
 
-			name := args[0]
 			factory.Printer = ch
 
-			cStack, err := cs.KpackClient.KpackV1alpha1().ClusterStacks().Get(name, metav1.GetOptions{})
```

Comment:
I guess so... will put it back when used twice. I remember adding it myself :) 
felt like the variable was frivolous, since the method calls where the name variable is used was documentation enough as to what args[0] was. (at least that was the thought process)

Reply:
it is pretty trivial 😂

Reply:
so, restore `name` or leave it as is? works fine either way for me.

Reply:
I say do `name` for extra clarity!

Reply:
restored all `name` I took out for args[0] with commit https://github.com/vmware-tanzu/kpack-cli/commit/07210d7966d200ba6173c72fb6bd62413d35f3ae
while doing it, noticed args[0] is the prevalent pattern across the repo.. not that I agree with it.

marking as resolved. please reopen if not ok